run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
+	conn, err := dbus.NewSystemConnection()
+	if err != nil {
+		return err
+	}
+
+	return runContainerInternal(containerID, containerPath, conn)
+}
+
+// Internal Functions
+
+// Produce a per-container unique name used to identify its transient systemd service
+func getUnitName(containerID string) string {
+	return fmt.Sprintf("kpod-container-%s.service", containerID)
```

Comment:
These are individual containers within a pod. I don't think we want to schedule an entire pod within a single systemd unit, we won't be able to track individual containers within the pod that way?